x-pack/test/reporting_api_integration/reporting_and_security/download_csv_dashboard.ts

```diff
@@ -25,7 +25,7 @@ export default function ({ getService }: FtrProviderContext) {
   const reportingAPI = getService('reportingAPI');
 
   const generateAPI = {
-    getCSVFromSearchSource: async (job: JobParamsDownloadCSV) => {
+    getCSVFromSearchSourceImmediate: async (job: JobParamsDownloadCSV) => {
```

Comment:
I changed this to include immediate since there are 3 types of csv export types. I can revert this if it's a problem.